src/ig/IGExporter.ts

```diff
@@ -1143,6 +1150,24 @@ export class IGExporter {
     }
   }
 
+  private getMetaExtensionDescription(resource: InstanceDefinition): string {
+    const description = resource.meta?.extension?.find(
+      e => e.url === 'http://hl7.org/fhir/StructureDefinition/instance-description'
+    )?.valueString;
```

Comment:
Based on the Zulip conversation, I think this is supposed to be `valueMarkdown`. See [zulip comment](https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Adding.20Description.20for.20Examples.20to.20ImplementationGuide.20-.20how/near/215415480) and related [jira issue](https://jira.hl7.org/browse/FHIR-29405).